observability/monitoring/common-monitoring-configuration-scenarios.adoc

```diff
@@ -0,0 +1,41 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="common-monitoring-configuration-scenarios"]
+= Common monitoring configuration scenarios
+include::_attributes/common-attributes.adoc[]
+:context: common-monitoring-configuration-scenarios
+
+toc::[]
+
+The default platform monitoring stack is available for use after installing {product-title}.
+The in-cluster stack includes the core platform Prometheus instance that collects metrics from your cluster and the core Alertmanager instance that  routes alerts.
+
+In addition to core platform monitoring, you can also optionally xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[enable monitoring for user-defined projects] for user workload monitoring.
+After doing so, users can then monitor their own services and workloads without the need for an additional monitoring solution.
+
+Depending on which users will use the monitoring stack and for what purposes, as a cluster administrator, you can configure monitoring components to suit the needs of different users in various scenarios.
+
+[id="monitoring-for-user-defined-projects-first-steps_{context}"]
+== Monitoring for user-defined projects: First steps
+
+As a cluster administrator, you can enable monitoring for user-defined projects in addition to core platform monitoring.
+Doing so enables non-administrator users such as developers to monitor their own projects outside of core platform monitoring.
+
+After you complete the following configuration activities, users can view collected metrics, query these metrics, and receive alerts for their own projects:
+
+* xref:../../observability/monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects[Enable user-defined projects.]
+* xref:../../observability/monitoring/enabling-monitoring-for-user-defined-projects.adoc#granting-users-permission-to-monitor-user-defined-projects_enabling-monitoring-for-user-defined-projects[Assign the `monitoring-rules-view`, `monitoring-rules-edit`, or `monitoring-edit` cluster roles] to grant non-administrator users permissions to monitor user-defined projects.
+* If needed, xref:../../observability/monitoring/enabling-monitoring-for-user-defined-projects.adoc#granting-users-permission-to-configure-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects[assign the `user-workload-monitoring-config-edit` cluster role] to grant non-administrator users permission to configure user-defined projects.
```

Comment:
```
$ oc -n openshift-user-workload-monitoring get role user-workload-monitoring-config-edit
NAME                                   CREATED AT
user-workload-monitoring-config-edit   2024-07-17T00:12:25Z
```
user-workload-monitoring-config-edit is role, not cluster role

Reply:
fixed